server/rest/src/main/java/org/infinispan/rest/resources/CacheManagerResource.java

```diff
@@ -196,21 +210,28 @@ public Invocations getInvocations() {
       NettyRestResponse.Builder responseBuilder = checkCacheManager(request);
       if (responseBuilder.getHttpStatus() == NOT_FOUND) return completedFuture(responseBuilder.build());
 
+      EmbeddedCacheManager withSubjectCM = cacheManager.withSubject(request.getSubject());
       Map<String, HealthStatus> cachesHeath = new HashMap<>();
+      // Remove internal caches
+      Set<String> cacheNames = new HashSet<>(withSubjectCM.getCacheNames());
+      cacheNames.remove(PROTOBUF_METADATA_CACHE_NAME);
+      cacheNames.remove(SCRIPT_CACHE);
```

Comment:
You can use `cacheNames.removeAll(internalCacheRegistry.getInternalCacheNames());`

Reply:
ok